.travis.yml

```diff
@@ -21,8 +21,15 @@ jdk:
   - oraclejdk7
   - oraclejdk8
 
+services:
+  - docker
+
+before_install:
+  - docker pull ibmcom/ibmjava:8-sdk
```

Comment:
I would prefer to use `library/ibmjava:8-jsk` because it's marked on dockerhub as "official" whereas `ibmcom` is just public. It's a bit curious though. It seems that they both point to the same place (repository) one points to "master" and one points to a githash.

`ibmcom/ibmjava:8-sdk` - https://github.com/ibmruntimes/ci.docker/blob/master/ibmjava/8/sdk/x86_64/ubuntu/Dockerfile

`library/ibmjava:8-sdk` - https://github.com/ibmruntimes/ci.docker/blob/a9a184211f9feef1f2bdd0330ff8ae95e6c015cf/ibmjava/8/sdk/x86_64/ubuntu/Dockerfile